File: pset6/mario.py
import cs50

def main():
    height = get_height()
    
    for i in range(height):
        #prints spaces
        print(" " * (height - (i + 1)), end = "")
        #prints hashes
        print("#" * (i + 2))

#to get the correct height    
def get_height():
    while True:
        print("Height: ", end = "")
        n = cs50.get_int()
        if (0 <= n <= 23):
            break
    return n
    
if __name__ == "__main__":
    main()

File: pset1/mario.c
#include<stdio.h>
#include<cs50.h>

int main(void)
{
    int height,i,space,hash;
    
    do
    {
        printf("\nheight: ");
        height = get_int();
    }while( (height < 0) || (height > 23) );
    
    for( i = height ; i > 0 ; i-- )
    {
        for( space = 1 ; space <= i-1 ; space++ )
        {
            printf(" ");
        }
        for( hash = 0 ; hash <= (height+1)-space ; hash++ )
        {
            printf("#");
        }
        printf("\n");
    }
    
    return 0;
}